### PR TITLE
added the ReIndex framework

### PR DESCRIPTION
**Pull request by dedalozzo**
I have added a new installer for the [ReIndex](https://github.com/dedalozzo/reindex) platform.

### CODE REVIEW

**Comment by dedalozzo** [comment]
Well, I'm losing hope, can you merge this please? Thank you

**Comment by niksamokhvalov** [comment]
Hi, write test, please. Example:
https://github.com/composer/installers/blob/master/tests/Composer/Installers/Test/InstallerTest.php#L87
https://github.com/composer/installers/blob/master/tests/Composer/Installers/Test/InstallerTest.php#L205

**Comment by dedalozzo** [comment]
I did it. 👍

File: tests/Composer/Installers/Test/InstallerTest.php [hunk 1]
```diff
@@ -270,6 +272,8 @@ public function dataForTestInstallPath()
             array('puppet-module', 'modules/puppet-name/', 'puppet/puppet-name'),
             array('redaxo-addon', 'redaxo/include/addons/my_plugin/', 'shama/my_plugin'),
             array('redaxo-bestyle-plugin', 'redaxo/include/addons/be_style/plugins/my_plugin/', 'shama/my_plugin'),
+            array('reindex-theme', 'themes/my_module/', 'author/my_module'),
+            array('reindex-plugin', 'plugin/my_module/', 'author/my_module'),
```
Comment (niksamokhvalov) [inline tests/Composer/Installers/Test/InstallerTest.php]: May be "plugin**s**"?

**Comment by niksamokhvalov** [comment]
Update your branch, please. This branch contains conflicts.

File: tests/Composer/Installers/Test/InstallerTest.php (hunk 1 above)
Comment (dedalozzo) [inline tests/Composer/Installers/Test/InstallerTest.php]: Yep, it's plugins actually, my mistake.

array('reindex-theme', 'themes/my_module/', 'author/my_module'),
array('reindex-plugin', 'plugins/my_module/', 'author/my_module'),

**Comment by dedalozzo** [comment]
Updated the branch and fixed the typo on the dir name.

**Comment by dedalozzo** [comment]
@niksamokhvalov I will really appreciate if you can merge this pull request. Thanks.

**Pull request merged**